taskcluster/ci/ui-test-apk/kind.yml

```diff
@@ -170,18 +170,96 @@ tasks:
                   key: firebaseToken
                   path: .firebase_token.json
                   json: true
+                - name: project/mobile/ci/testrail
+                  key: testrailCredentials
+                  path: .testrail_credentials.json
+                  json: true
             pre-commands:
                 - ["cd", "focus-android"]
             commands:
                 - [wget, {artifact-reference: '<signed-apk-debug-apk/public/build/target.arm64-v8a.apk>'}, '-O', app.apk]
                 - [wget, {artifact-reference: '<signed-apk-android-test/public/build/target.noarch.apk>'}, '-O', android-test.apk]
                 - ['automation/taskcluster/androidTest/ui-test.sh', 'arm-beta-tests', 'app.apk', 'android-test.apk', '1']
+                - [python3, automation/taskcluster/androidTest/test_env.py]
```

Comment:
With every push to a `releases` branch this would be executed, is this what we want?

Reply:
if Soft Vision folks are running test automation on each push, then yes, as it will create their milestones and test runs for them each time. 

if they don't run test automation each time, then we need to understand why not and when we want to do that instead.

i'm assuming that's what we wanted, but that only because i'm assuming they run the tests after each push (if a push represents a bug fix, for example).

Reply:
No, we want them to run only when a release build is shipped to Taskcluster, we don't need to test every uplift on that branch.